restapi/Controllers/TimesheetsController.cs

```diff
@@ -190,6 +190,12 @@ public IActionResult Submit(Guid id, [FromBody] Submittal submittal)
                     return StatusCode(409, new EmptyTimecardError() { });
                 }
 
+                // Timecard employee cannot submit other's timecard
+                if(timecard.Employee != submittal.Person)
+                {
+                    return StatusCode(409, new InvalidStateError() { });
```

Comment:
might be better as an authorization error. ie, "you don't have access to do this thing"